src/core/profiles/messages.ts

```diff
@@ -4,4 +4,12 @@ export const MY_PROFILE_SUCCESFULLY_UPDATED =
   'Tu perfil se ha actualizado exitosamente!';
 export const PROFILE_SUCCESFULLY_DELETED_SKILL =
   'Skill eliminado de tu perfil exitosamente';
+export const PROFILE_SUCCESFULLY_DELETED_LANGUAGE =
+  'Idioma eliminado de tu perfil exitosamente';
 export const ERROR_PROFILE_SKILL_NOT_FOUND = 'Perfil o skill no se encontraron';
+export const ERROR_PROFILE_LANGUAGUE_NOT_FOUND =
+  'Perfil o idioma no se encontraron';
```

Comment:
¿Por que Perfil o idioma? el perfil tiene otra validacion anterior con otro mensaje si no m equivoco
```suggestion
export const ERROR_LANGUAGUE_NOT_FOUND =
  'El idioma que intentas agregar al perfil no existe';
```